router/auth.go

```diff
@@ -239,6 +240,22 @@ func authInjectYellowfin(log *log.Logger, w http.ResponseWriter, req *http.Reque
 		defer logoutResp.Body.Close()
 
 		return false // do not forward this to YF
+
+	// The front-end with an open Iframe to YF will keep polling to check if it is
+	// the session currently allowed to access YF. If another instance is opened with the same user,
+	// that instance will get access, and the first Iframe should exit and go to IMQS home page.
+	case "/yellowfin/checksession":
+		target.lock.RLock()
+		token_yf, exists := target.tokenMap[authData.Identity].(*yellowfinToken)
+		if !exists || token_yf.UserSession == "" {
+			fmt.Fprintf(w, "%s", "No Session Found")
```

Comment:
I'm actually using something like `if (iq.readCookie(iq.session.imqsCookieName) !== res.responseText)` in the front-end. (https://github.com/IMQS/www/pull/999)

If I return something else than 200 or 201 then the front-end will have to return to the IMQS homepage when the ajax call hits the onError callback. I'm unsure if this is better?